CHANGELOG.md

```diff
@@ -13,12 +13,22 @@
 * XCAssets: the `colorAliasName` and `imageAliasName` template parameters are now deprecated and will be removed in the next major release.  
   [David Jennes](https://github.com/djbe)
   [#614](https://github.com/SwiftGen/SwiftGen/pull/614)
+  
+### Deprecations
+
+* The subcommand `swiftgen templates` has been renamed `swiftgen template` (singular); the plural form of the command has been deprecated and will be removed in next major version.  
+  [@AliSoftware](https://github.com/AliSoftware)
+  [#697](https://github.com/SwiftGen/SwiftGen/pull/697)
+
 
 ### New Features
 
 * You can now easily create a new config file using `swiftgen config init`. This will create an example and commented config file and open it to let you edit it to your needs.  _Note that the generated config file is static content which doesn't take the user's project into account (though that might change in the future)_.  
   [@AliSoftware](https://github.com/AliSoftware)
   [#694](https://github.com/SwiftGen/SwiftGen/pull/694)
+* You can now use `swiftgen template doc [parser] [template]` on the command line to quickly open the documentation for templates on GitHub direclty from your terminal.  
```

Comment:
Should that be `templateName`? Just to not confuse users that they can use it for custom templates. Maybe I'm worrying too much 😅

Reply:
Yeah good point, worth being more specific.

```suggestion
* You can now use `swiftgen template doc [parser] [templateName]` on the command line to quickly open the documentation for templates on GitHub direclty from your terminal.  
```

NB: I think it's enough to be more specific in CHANGELOG only but not change the code though, as if you try to run `swiftgen template doc xcassets my-custom-template` then the implementation checks for bundled templates only (doesn't consider templates from Application Support nor custom templates) and exits with explicit error if template name not found.